eclair-core/src/main/scala/fr/acinq/eclair/db/BackupHandler.scala

```diff
@@ -0,0 +1,87 @@
+package fr.acinq.eclair.db
+
+import java.io.File
+
+import akka.actor.{Actor, ActorLogging, Status}
+import akka.pattern.pipe
+import fr.acinq.eclair.channel.ChannelPersisted
+import grizzled.slf4j.{Logger, Logging}
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class BackupHandler(databases: Databases, wip: File, destination: File)(implicit ec: ExecutionContext) extends Actor with ActorLogging {
+  import BackupHandler._
+
+  // we listen to ChannelPersisted events, which will trigger a backup
+  override def preStart(): Unit = {
+    super.preStart()
+    context.system.eventStream.subscribe(self, classOf[ChannelPersisted])
```

Comment:
This may be better to suscribe to events here, but we do it differently in all other actors. Maybe stay consistent for now and do another PR to square all actors up at the same time?